libexec/bootstrap-scripts/deffile-driver-localimage.sh

```diff
@@ -46,9 +46,7 @@ fi
 umask 0002
 
 message 1 "Exporting contents of ${FROM} to ${SINGULARITY_IMAGE}\n"
-
-cmd="${SINGULARITY_libexecdir}/singularity/bin/export ${FROM} | (cd ${SINGULARITY_ROOTFS} && tar xBf -)"
-if ! eval $cmd; then
+if ! eval "${SINGULARITY_bindir}"/singularity image.export "${FROM}" | (cd "${SINGULARITY_ROOTFS}" && tar xBf - -C "${SINGULARITY_ROOTFS}"); then
```

Comment:
I believe we can get rid of the `cd ...` now that we are using the `-C` option to tar.